scripts/test-staged.js

```diff
@@ -14,9 +14,12 @@ const { execSync } = require('child_process');
 // find names of staged files
 const stagedFiles = execSync('git diff --cached --name-only --diff-filter=ACMR')
```

Comment:
@Willie-The-Lord, you are not adding a D in the filter as suggested by @BSFishy. Check the comment [here](https://github.com/opensearch-project/oui/issues/1071#issuecomment-1755752789)

Reply:
@BigSamu Since your issue is happening in OUI version 1.3, this PR is specifically for version 1.3. The issue is resolved in the the latest version. Please checkout [this PR](https://github.com/opensearch-project/oui/pull/1072). If you still run into any problem, please let me know!

Reply:
Baaaaa, sorry you right @Willie-The-Lord. I forgot this problem was from OUI v1.3. However, I am still getting the same errors in the PR you did. Is it working correctly from your side? Also, did you check my comment above for adding the D filter as @BSFishy suggested?

Reply:
@BigSamu Lemme check on my end for the error. Regarding the `-D` flag, I've provided the reason why the original code does not include`-D` in the previous comment. It seems reasonable to me. But welcome any other suggestion why adding `-D` is necessary.

Reply:
You right! Make sense in your comment above to not include the -D flag! OK, I wait for your check on your end.

Reply:
Ah! I think that the issue with your console output isn't related to `scripts/test-staged.js`. The problem arises because the dependencies in OUI v1.3 differ from those in the latest OUI version. Could you please run `yarn` in your terminal again, and then reproduce the following steps `git add . && git commit -sm "test.txt"`?